src/install.c

```diff
@@ -934,10 +946,19 @@ static gboolean pre_install_checks(gchar* bundledir, GPtrArray *install_plans, G
 		}
 
 skip_filename_checks:
-		if (!g_file_test(plan->target_slot->device, G_FILE_TEST_EXISTS)) {
-			g_set_error(error, R_INSTALL_ERROR, R_INSTALL_ERROR_NODST,
-					"Destination device '%s' not found", plan->target_slot->device);
-			return FALSE;
+		if (g_strcmp0(plan->target_slot->type, "boot-nor-fallback") == 0) {
+			devices = g_strsplit(plan->target_slot->device, ":", -1);
```

Comment:
This `g_strsplit` should not be at multiple places. Instead, the handling for multiple devices should be explicit in the `RaucSlot` struct and should be split during parsing in `config_file.c`. For consistency, it should use `g_key_file_get_string_list`, which splits at `;`. If only one device is give, it should behave as before, storing it in the `gchar *device`. Otherwise a new `gchar **devices` would be used and device would be `NULL`.

This way, it's explicit if one or multiple devices are configured. Then, we'd need a consistency check at the end of `load_config()` which ensures that only `boot-nor-fallback` passes two devices. All others would need only one.